core/src/test/java/org/apache/calcite/sql/test/SqlOperatorBaseTest.java

```diff
@@ -5577,6 +5577,46 @@ private void checkNullOperand(SqlTester tester, String op) {
     }
   }
 
+  @Test public void testLastDayFunc() {
```

Comment:
do you need an end to end test to verify this new function in a SQL query?

Reply:
if so, add a few queries to `misc.iq`

Reply:
Will do. Thanks.